src/cunumeric/fft/fft_util.h

```diff
@@ -0,0 +1,104 @@
+/* Copyright 2021 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "cunumeric/cunumeric.h"
+
+namespace cunumeric {
+
+using namespace legate;
+
+// These fft types match cufftType
+enum class fftType {
+  FFT_R2C = 0x2a,  // Real to complex (interleaved)
```

Comment:
So this enum is decoupled from the enum in the Python code, but I think you should use the same enum to avoid any mistakes. Please look at how `CuNumericOpCode` is written.